src/pages/derivx/_start-derivx.tsx

```diff
@@ -283,9 +283,11 @@ const StartDerivX = () => {
                         return (
                             <SideTab.Panel
                                 key={index}
```

Comment:
do we really need this change? remove 3 props and transfer to an object

Reply:
`const { label, description, item_width, mobile_item_width } = child`

use this approach to fix your problem, then you don't need to transfer the props

Reply:
Thank u 
[@Eduard](https://deriv-group.slack.com/team/U03GC37S4R4)
 for your revision, yes you are right it's better to change the props of the component,
I will change this :blush: